### ISSUE TITLE
Using the Disabler causing massive 

### ISSUE THREAD

**Issue by Umbra999** [jbara2002/windows-defender-remover#57]
When i use the defender on Win 11 22h2 (Build 22621.1778) i get massive lags on my System, i tried it on multiple Systems just to be sure. for example i open a exe and the exe takes 30 seconds to open instead of opening instant, when i enable everything again the issue is gone. Also removing the defender completly does not have this issue

**Comment by jbara2002** [comment]
Noted. Planned to fix

**Comment by stimpy81** [comment]
> When i use the disabler on Win 11 22h2 (Build 22621.1778) i get massive lags on my System, i tried it on multiple Systems just to be sure. for example i open a exe and the exe takes 30 seconds to open instead of opening instant, when i enable everything again the issue is gone. Also removing the defender completly does not have this issue

This is caused by SmartScreen, I have the same issue.  Jbara has posted a reg command to help in this forum.

**Comment by Umbra999** [comment]
> > When i use the disabler on Win 11 22h2 (Build 22621.1778) i get massive lags on my System, i tried it on multiple Systems just to be sure. for example i open a exe and the exe takes 30 seconds to open instead of opening instant, when i enable everything again the issue is gone. Also removing the defender completly does not have this issue
> 
> This is caused by SmartScreen, I have the same issue. Jbara has posted a reg command to help in this forum.

oh really? which reg command is it?

**Comment by Umbra999** [comment]
> Noted. Planned to fix

nvm, the issue seems to be solved by disabling "Smart App Control", lag is gone

**Comment by stimpy81** [comment]
Unfortunately, the reg keys and method for installing them didn't work.

**Comment by jbara2002** [comment]
yes, the smartscreen feature is f*cking at all. in Windows 10 is not producing any problems.. in windows 11 is producing problems with uwps, lag and all. When MSFT is introducing drivers which is used for security guards and idk, easy to remove/disable. i think, i will found another solutions for disabling smartscreen without harm system. (Disabling via policy is a good ideea, but its not ok for maintenance task which is modifing that configuration.)
😇 the best part is the script is not need to disable tamper protection to apply the script (remove , disable , enable) it auto disable alongside with defender components.
